### ISSUE TITLE
CALL FUNCTION DESTINATION - unused_variables - false positive

### ISSUE THREAD

**Issue by Zeatynis** [abaplint/abaplint#3935]
```
REPORT zfoo.
DATA rfc_dest TYPE string VALUE `SOME_NAME`.

CALL FUNCTION 'RFC_FUNCTION' DESTINATION rfc_dest.
```

/zfoo.prog.abap[2, 6]: Variable "rfc_dest" not used(unused_variables)

Similar to #3930

**Comment by larshp** [comment]
thanks

**Comment by larshp** [comment]
@Zeatynis think this is fixed?

<img width="1157" height="765" alt="Image" src="https://github.com/user-attachments/assets/c86e2958-1c0a-430f-b9aa-958acb3bd046" />

**Comment by larshp** [comment]
closing
